docs/products/postgresql/howto/use-pg-audit-logging.md

```diff
@@ -0,0 +1,337 @@
+---
+title: Collect audit logs in Aiven for PostgreSQL®
+sidebar_label: Collect audit logs
+---
+
+import AuditLogsOpenSearchDashboards from "@site/static/images/content/products/postgresql/pgaudit-logs-in-os-dashboards.png";
+
+Enable and configure the [Aiven for PostgreSQL® audit logging feature](/docs/products/postgresql/concepts/pg-audit-logging) on your service. Access and visualize your logs to monitor activities on your databases.
+
+## Prerequisites
+
+- PostgreSQL version 11 or higher
+- `avnadmin` superuser role
+- Dev tool of your choice to interact with the feature
+  - [Aiven Console](https://console.aiven.io/)
+  - [Aiven CLI client](/docs/tools/cli)
+  - [psql](https://www.postgresql.org/docs/current/app-psql.html) for advanced
+    configuration
+
+## Enable audit logging
+
+Enable audit logging by setting the `pgaudit.feature_enabled` parameter to
+`true` in your service's advanced configuration. Using the Aiven
+[console](https://console.aiven.io/), [CLI](/docs/tools/cli), or
+[psql](https://www.postgresql.org/docs/current/app-psql.html).
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs>
+<TabItem value="1" label="Aiven Console" groupId="group1" default>
+### Enable in Aiven Console
+
+:::important
+In [Aiven Console](https://console.aiven.io/), you can enable audit logging at the service
+level only. To enable it on a database or for a user, use
+[psql](https://www.postgresql.org/docs/current/app-psql.html).
+:::
+
+1. Log in to [Aiven Console](https://console.aiven.io/), and go to your organization
+   \> project > Aiven for PostgreSQL service.
+1. On the **Overview** page of your service, select **Service settings** from the sidebar.
+1. On the **Service settings** page, go to the **Advanced configuration** section
+   and select **Configure**.
+1. In the **Advanced configuration** window, select **Add configuration options**, add
+   the `pgaudit.feature_enabled` parameter, set it to `true`, and select
+   **Save configuration**.
+</TabItem>
+<TabItem value="2" label="Aiven CLI">
+### Enable with Aiven CLI
+
+Use the [Aiven CLI client](/docs/tools/cli) to run the
+[avn service update](/docs/tools/cli/service-cli) command. Update your service by setting
+the `pgaudit.feature_enabled` parameter's value to `true`.
+
+```bash
+avn service update -c pgaudit.feature_enabled=true SERVICE_NAME
+```
+
+:::important
+By default, audit logging does not emit any audit records. To trigger a logging operation
+and start receiving audit records, configure audit logging parameters as detailed in
+[Configure audit logging](#configure-audit-logging).
+:::
+</TabItem>
+<TabItem value="3" label="psql">
+### Enable with psql
+
+:::note
+psql allows for fine-grained enablement of audit logging: on a database, for a user (role),
```

Comment:
The "user (role)" part is a little confusing to me. Will the logs show an individual service user's activity or that of a role?